src/bci_build/package/registry/README.md.j2

```diff
@@ -6,8 +6,7 @@ This container image allows to run a local OCI registry.
 
 
```

Comment:
Should the changes in this file go in? Looks like an accidental/unrelated change

Reply:
no, this is unrelated. weird.